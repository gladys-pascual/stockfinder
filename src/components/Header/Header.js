import { Link } from "react-router-dom";
import "./Header.scss";

const Header = () => {
  return (
    <header>
      <Link exact="true" to="/" className="logo">
        <img src="https://via.placeholder.com/50" alt="Logo of StockFinder" />
      </Link>
    </header>
  );
};

export default Header;
