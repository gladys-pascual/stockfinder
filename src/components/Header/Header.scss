@import "../../assets/variables.scss";

header {
  display: flex;
  .logo {
    width: 150px;
    img {
      width: 100%;
    }
  }
  .logo:hover {
    animation-name: logo-bounce-animation;
    animation-duration: 1s;
  }
}
