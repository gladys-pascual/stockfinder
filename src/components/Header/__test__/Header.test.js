import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import Header from "../Header";

test("renders Header component as expected", () => {
  const { container } = render(
    <BrowserRouter>
      <Header />
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});
