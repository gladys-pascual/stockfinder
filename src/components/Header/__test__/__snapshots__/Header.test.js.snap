// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header component as expected 1`] = `
<div>
  <header>
    <a
      class="logo"
      exact="true"
      href="/"
    >
      <img
        alt="Logo of StockFinder"
        src="StockFinderLogo.svg"
      />
    </a>
    <nav>
      <ul>
        <li>
          <a
            aria-current="page"
            class="header-link selected"
            href="/"
          >
            Home
          </a>
        </li>
        <li>
          <a
            class="header-link"
            href="/marketnews"
          >
            Market News
          </a>
        </li>
      </ul>
    </nav>
  </header>
</div>
`;
